### PR TITLE
Sign Out Functionality

### PR DESCRIPTION
**Pull request by sam-kakengi**
Sign out functionality added to the userDrawer button.

### CODE REVIEW

File: frontend/src/components/core-components/dashboard/UserDrawer.js [hunk 1]
```diff
@@ -75,7 +84,7 @@ const UserDrawer = ({ drawerOpen, toggleDrawer, userInfo }) => {
                     />
                         
                         {/* Logout */}
-                    <ListItem onClick={() => console.log('Logout clicked')}>
+                    <ListItem onClick={signOut}>
```
Comment (Dimwiddle) [inline frontend/src/components/core-components/dashboard/UserDrawer.js:87]: Will this also end the auth session?

In other words - will the user have to login again?

File: backend/running/doodle.py
[the remark this answers is not in the document]
Comment (sam-kakengi) [reply backend/running/doodle.py:1]: Done! Had it in gitignore but that was on the other branch haha.

File: frontend/src/components/core-components/dashboard/UserDrawer.js (hunk 1 above)
Comment (Dimwiddle) [reply frontend/src/components/core-components/dashboard/UserDrawer.js:87]: OK great - less code needed than I thought 👌

**Review by Dimwiddle — APPROVED** [review]

**Comment by Dimwiddle** [comment]
Looks good! Ready to merge whenever you're ready.

Not sure if the other PR will cause merge conflicts... maybe just try merge this first to see haha.

**Pull request merged**